scripts/docker/ubuntu/xenial/Dockerfile

```diff
@@ -56,4 +58,21 @@ RUN useradd \
     --shell "/bin/bash" \
     jenkins
 
+# Handle Haskell dependencies
+# We use ghc8.2.2 from the haskell platform website here, as xenial only provides ghc7
+RUN wget https://haskell.org/platform/download/8.2.2/haskell-platform-8.2.2-unknown-posix--core-x86_64.tar.gz \
```

Comment:
I assume building Haskell from source will take quite some time. Would it maybe make sense to use the [personal package archive (PPA) for GHC of Herbert V. Riedel](https://launchpad.net/~hvr/+archive/ubuntu/ghc) instead? Since the [official Haskell homepage](https://www.haskell.org/ghc/distribution_packages.html#ubuntu) als mentions this PPA, I guess the support for this third party repo should be quite good.

Reply:
this is the dockerfile so it will be only built once, imo that shouldn't make a big difference.
This branch should mostly show whether its possible on xenial in general and doesn't directly resemble what piankero did no.

Reply:
this is the dockerfile so it will be only built once, imo that shouldn't make a big difference.
This branch should mostly show whether its possible on xenial in general and doesn't directly resemble what piankero did no.